ratpack-core/src/main/java/ratpack/handling/ByContentSpec.java

```diff
@@ -78,7 +78,7 @@
    * @param block the code to invoke if the content type matches
    * @return this
    */
-  default ByContentSpec type(String mimeType, Block block) {
+  default ByContentSpec type(CharSequence mimeType, Block block) {
```

Comment:
This works ok because `String` is a `CharSquence`, right? We aren't breaking any compatability?

Reply:
This breaks binary compatibility, but not source compatibility. It needs to be added as an overload.